services/blockchainService.ts

```diff
@@ -24,13 +22,30 @@ export interface GetAddressTransactionsParameters {
   maxTransactionsToReturn: number
 }
 
+interface InputOutput {
+  value: number
+  outputScript?: string
```

Comment:
another question: I see that rn chronik is using the outputScript and `grpc` the `address`. Isn't there a way to convert the `outputScript` to `address` so we only use one field and make it required?

Reply:
yes indeed! Damn, much better, done